superset-frontend/src/dashboard/util/activeAllDashboardFilters.ts

```diff
@@ -92,7 +92,7 @@ export const getAllActiveFilters = ({
     const scope = nativeFilters?.[filterId]?.scope ??
       chartConfiguration?.[filterId]?.crossFilters?.scope ?? {
         rootPath: [DASHBOARD_ROOT_ID],
-        excluded: [],
+        excluded: [filterId],
```

Comment:
Is there risk that `chartConfiguration[filterId].crossFilters.scope.excluded == []`, in which case the filter won't exclude itself?

Reply:
I haven't encountered such case. It's either an empty object or an array with a chart ID.